src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,45 @@
+package pages;
+
+import enums.BrowserName;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+
+public class LoginPage {
+
+    private BasePage basePage;
+
+    // Locators:
+    public By registerAccountButtonLocator = By.id("accountFrm_accountregister");
+    public By guestCheckoutButtonLocator = By.id("accountFrm_accountguest");
+    public By loginInputLocator = By.id("loginFrm_loginname");
+    public By passwordInputLocator = By.id("loginFrm_password");
+    public By loginButtonLocator = By.xpath("//*[@id='loginFrm']/button");
+
+    public LoginPage(BrowserName browserName) {
+    }
+
+    // Methods:
+    public void selectRegisterAccountOption() {
+        basePage.isElementDisplayed(registerAccountButtonLocator);
+    }
+
+    public void selectGuestAccountOption() {
+        if (basePage.isElementDisplayed(guestCheckoutButtonLocator)) {
```

Comment:
We don't need this exception in the tests. I would rather have method with parameter type boolean to check if this element is hidden or not and for it to be a part of some test.

Reply:
@Fishgame1 Method with boolean parameter added to new test 'TestGuestCheckout'.

Reply:
So can we remove that exception from here?:)

Reply:
@Fishgame1 As per previous comments, I think it'd be best to fix it as part of basic tests rework - not only would I have to remove the exception from `LoginPage` class, I'd also use the methods provided in `BasePage` class for this